src/views/Scenario/Scenario.js

```diff
@@ -228,7 +243,29 @@ const Scenario = () => {
     label: scenarioListLabel,
     validationStatus: scenarioValidationStatusLabels,
   };
+  // FOR DEVELOPMENT PURPOSE ONLY, DELETE BEFORE MERGE
+  const lastRun = '2023-03-13T16:25:30.795486885Z';
+  // FOR DEVELOPMENT PURPOSE ONLY, DELETE BEFORE MERGE
 
+  const hasScenarioBeenRun = useMemo(
+    () => currentScenario?.data?.lastRun !== null && currentScenario?.data?.state === 'Successful',
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Do we still need the exception `eslint-disable-next-line`? Idem for the next `useMemo` below